assets/src/customizer/components/google-fonts-control/index.js

```diff
@@ -43,6 +44,7 @@ const GoogleFontsControl = props => {
 	const [ isExpanded, setIsExpanded ] = useState( false );
 	const [ items, setItems ] = useState( children );
 	const [ selectedFont, setSelectedFont ] = useState( value );
+	const [ setRequesting ] = useState( false );
```

Comment:
`useState` usually works with two variables, in this case `requesting` is missing.

```suggestion
	const [ requesting, setRequesting ] = useState( false );
```